app/League-Entries/page.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import { LeagueEntries } from '@/components/LeagueEntries/LeagueEntries';
+import { LeagueSurvivors } from '@/components/LeagueSurvivors/LeagueSurvivors';
+import * as React from 'react';
+
+/**
+ * @returns Leagues
+ */
+const Leagues: React.FC = (): React.JSX.Element => {
+  return (
+    <div className="Leagues mx-auto max-w-3xl pt-10">
```

Comment:
Always match the variable name and classNames to the component or page name.
```suggestion
    <div className="league-entries mx-auto max-w-3xl pt-10">
```

Reply:
Updated in latest commit.